### PR TITLE
update cursorIterator is change in ArrayIterator

### PR DESCRIPTION
**Pull request by itkg-alavieille**
in relation with https://github.com/open-orchestra/open-orchestra-model-bundle/pull/327

### CODE REVIEW

**Comment by nicolasThal** [comment]
:+1:

File: ApiBundle/Controller/NodeController.php [hunk 1]
```diff
@@ -93,7 +93,7 @@ public function deleteAction($nodeId)
     {
         $siteId = $this->get('open_orchestra_backoffice.context_manager')->getCurrentSiteId();
         $nodes = $this->get('open_orchestra_model.repository.node')->findByNodeIdAndSiteId($nodeId, $siteId);
-        $node = $nodes->getNext();
+        $node = next($nodes);
```
Comment (nicolasThal) [inline ApiBundle/Controller/NodeController.php]: you could use first?

**Comment by itkg-alavieille** [comment]
@nicolasThal

**Comment by nicolasThal** [comment]
:+1:

**Pull request merged**